Src/Framework/Jobs/Castep.py

```diff
@@ -231,7 +231,7 @@ def run_step(self, index):
 
         # Retrieve the forces multiplied by Units.Hartree/Units.Bohr and save them
         self._forces.array = config[2 * nAtoms:3 * nAtoms, :]
-        data["forces"] = self._forces
+        data["gradients"] = self._forces

```

Comment:
Hi, are you certain this should be gradients? My understanding of MMTK is that this variable denotes the energy gradients, not forces.

Reply:
Hi both. From what I see, in MMTK "gradients" stands for the energy gradients at each atom position and as @eurydice76 says, the force is the negative gradient of the potential energy. From this point of view, I would imagine that data["gradients"] = -self._forces. But do we use "gradients" in any analysis? And "forces"? Can we have both and do we need both?

Reply:
I see, thank you for the answer! To the best of my knowledge, no current analysis uses gradients or forces. Also, we technically cannot have a variable called 'forces' since MMTK does not recognise such variable, and so essentially does not write it into the NetCDF trajectory.

Reply:
Thtats' why I turned forces to gradients in order that MMTK can "recognize" them. However, as you said Rastislav, no analysis uses the force/gradient. So we should perhaps not worry too much about this right now.